### PR TITLE
export: Sync JSON fields with CSV and TSV, use NA for default SGTF

### PR DESCRIPTION
**Pull request by abhidg**
Fixes: #2354

### CODE REVIEW

File: data-serving/scripts/export-data/test_transform_mongoexport_expected.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
...
...umber": "0", "symptoms.status": "", "symptoms.values": "", "transmission.linkedCaseIds": "", "transmission.places": "", "transmission.routes": "", "travelHistory.travel.dateRange.end": "", "travelHistory.travel.dateRange.start": "", "travelHistory.travel.location.administrativeAreaLevel1": "", "travelHistory.travel.location.administrativeAreaLevel2": "", "travelHistory.travel.location.administrativeAreaLevel3": "", "travelHistory.travel.location.country": "", "travelHistory.travel.location.geoResolution": "", "travelHistory.travel.location.geometry.coordinates": "", "travelHistory.travel.location.name": "", "travelHistory.travel.location.place": "", "travelHistory.travel.methods": "", "travelHistory.travel.purpose": "true", "travelHistory.traveledPrior30Days": "", "vaccines.0.batch": "", "vaccines.0.date": "", "vaccines.0.name": "", "vaccines.0.sideEffects": "", "vaccines.1.batch": "", "vaccines.1.date": "", "vaccines.1.name": "", "vaccines.1.sideEffects": "", "vaccines.2.batch": "", "vaccines.2.date": "", "vaccines.2.name": "", "vaccines.2.sideEffects": "", "vaccines.3.batch": "", "vaccines.3.date": "", "vaccines.3.name": "", "vaccines.3.sideEffects": null, "variantOfConcern": ""}
```
Comment (jim-sheldon) [inline data-serving/scripts/export-data/test_transform_mongoexport_expected.json:3]: This needs a closing `]`.

Comment (jim-sheldon) [reply data-serving/scripts/export-data/test_transform_mongoexport_expected.json:3]: `genomeSequences` should not be present.

Comment (abhidg) [reply data-serving/scripts/export-data/test_transform_mongoexport_expected.json:3]: Are we removing genomeSequences entirely from the output? Then I'll update the data dictionary as well.

Comment (jim-sheldon) [reply data-serving/scripts/export-data/test_transform_mongoexport_expected.json:3]: Yeah, that was discussed and changed in https://github.com/globaldothealth/list/pull/2355

**Review by iamleeg — APPROVED** [review]

**Review by jim-sheldon — APPROVED** [review]

**Pull request merged**